test/dotnet-test.Tests/project.json

```diff
@@ -19,7 +19,7 @@
     "System.Net.Sockets": "4.1.0-rc2-24015",
     "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24015",
     "xunit": "2.1.0",
-    "dotnet-test-xunit": "1.0.0-dev-140469-38"
+    "dotnet-test-xunit": "1.0.0-rc2-152011-55"
```

Comment:
Yup. I need to check in dotnet-test-xunit for it to product a build. This was a locally produced build which wouldn't be the one we'd be using.